app/services/transaction_search/order_status_searcher.rb

```diff
@@ -3,10 +3,9 @@ module TransactionSearch
   class OrderStatusSearcher < BaseSearcher
 
     def options
-      # Unlike the other lookups, this query is much faster this way than using a subquery
-      OrderStatus.find_by_sql(order_details.joins(:order_status)
-                                                         .select("distinct(order_statuses.id), order_statuses.facility_id, order_statuses.name, order_statuses.lft")
-                                                         .reorder("order_statuses.lft").to_sql)
+      OrderStatus.select("order_statuses.id, order_statuses.facility_id, order_statuses.name, order_statuses.lft")
```

Comment:
This is probably fine enough. I worry about the comment around the performance of the query, though. If we do go this way, probably don't need the `.select("order_statuses.id, order_statuses.facility_id, order_statuses.name, order_statuses.lft")`, and we can just select everything. OrderStatus doesn't have a lot of cruft on it.

Reply:
Admittedly I don't have massive amounts of seed data I'm testing against, but I'm not seeing any noticeable slowness, and this is demonstrably better memory-wise. We can always optimize for speed later if it turns out to be an issue.

Reply:
I think that's fine. My only concern is that we did optimize this at one point. Let me do a quick test again NU data and make sure this doesn't cause a big impact.